### DISCUSSION TITLE
readme.md update

### DISCUSSION THREAD

**Discussion by maharanasarkar** [maharanasarkar/whatsapp-connector-rasa#10] [Announcements]
Hey folks,
I would be updating the readme.md file soon, guide to building rasa from source with the whatsapp connector will be mentioned in the file for both linux and windows.
Kindly allow me some time.

Thanks
Maharana

**Comment by mumbaigirl** [comment]
@maharanasarkar The readme.md file is already okay to get-started, could you kindly do same for Instagram-Rasa Integration please.  Also would be obliged to you if you could please show **how to send interactive messages from RASA to WhatsApp?**.
I connected it to Rasa , it's working very good for static text messages.
I just can't get interactive messages working. You seem to have excellent knowledge about both Rasa and Meta Integration , that's how you came up with  such a beautiful piece of code. Please share you knowledge here.  Please solve Instagram Integration issue also.

**Reply by maharanasarkar** [reply]
@thatcuteseller to use interactive buttons you need to get your message template get approved from meta to be used. Also for dynamic messages it gets difficult to get it approved.

**Reply by maharanasarkar** [reply]
You can look up some forums or meta documentation for the interactive buttons. I personally used twillio for production because it becomes a mediator to get our message template to be sent to meta for approval and I've seen high success rate from twillio platform even for dynamic message

**Reply by mumbaigirl** [reply]
@maharanasarkar I have a verified business profile , I can get the templates approved as I am already using WhastApp Business Cloud API. However, to be honest with you, I don't know how to send interactive messages from RASA to Whatsapp, I know , I am doing it in a wrong way as I am using ChatGPT.  Hence requesting you , if you could please help. 

Second thing is , I want to mark the incoming messages as '**seen**' , but , I don't know how to go about it?

3rd , I could not get Instagram connected , I followed this [repo for instagram integration](https://github.com/rafaelinacioo/rasa_instagram).  Looks like FB Meta has changed the API's or something.
![Screenshot 2024-03-11 184258](https://github.com/maharanasarkar/whatsapp-connector-rasa/assets/99070985/d728e9eb-b373-42ec-a251-b6c76125aa00)


But, after integrating your code , I felt so confident that you have core knowledge about both platforms , RASA as well as Meta. Your WhatsApp code worked perfectly!

I am trying to solve these issues from past 3-4 days :(
